src/RoverPhoto/RoverTopPhoto.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import './RoverTopPhoto.scss';
+import '../Fonts.scss';
```

Comment:
what do we need these for?

Reply:
Sorry this is supposed to be on the ../Colours.scss import, I think that one can go